arangod/RestHandler/RestReplicationHandler.cpp

```diff
@@ -1508,6 +1509,12 @@ Result RestReplicationHandler::parseBatch(std::string const& collectionName,
         Result res =
             restoreDataParser(ptr, pos, collectionName, line, key, builder, doc, type);
         if (res.fail()) {
+          if(res.errorNumber() == TRI_ERROR_HTTP_CORRUPTED_JSON) {
+            using namespace std::literals::string_literals;
+            auto data = std::string(ptr, pos);
+            LOG_TOPIC("f86f3", ERR, Logger::RESTORE) << "invalid json in: " + data;
```

Comment:
I think we can remove the log message here, as long as a proper error message will make it to the end user (and I think it will, as the error message is in `res`, which will be returned).